packages/core/src/Configs/StrategyConfig.class.ts

```diff
@@ -1,41 +1,44 @@
 import { AssetAmount, IAssetAmountMetadata } from "@sundaeswap/asset";
 
-import { IStrategyConfigArgs, TOrderAddresses } from "../@types/index.js";
+import {
+  IPoolData,
+  IStrategyConfigArgs,
+  TOrderAddresses,
+} from "../@types/index.js";
 import { Config } from "../Abstracts/Config.abstract.class.js";
+import { OrderConfig } from "../Abstracts/OrderConfig.abstract.class.js";
 
 /**
  * The main config class for building valid arguments for listing a strategy order.
  */
-export class StrategyConfig extends Config<IStrategyConfigArgs> {
+export class StrategyConfig extends OrderConfig<IStrategyConfigArgs> {
   suppliedAssets?: AssetAmount<IAssetAmountMetadata>[];
-  orderAddresses?: TOrderAddresses;
+
+  ownerPublicKey?: string;
```

Comment:
is the ownerPublicKey the authorization key? if so, lets avoid the word `owner`;

instead, lets use the term `executorPublicKey`, to emphasize that it can (and usually will be) different from the owner.